xwiki-platform-core/pom.xml

```diff
@@ -164,6 +164,14 @@
                   <code>java.method.removed</code>
                   <old>method java.lang.String com.xpn.xwiki.doc.XWikiDocument::displayTooltip(java.lang.String, com.xpn.xwiki.objects.BaseObject, com.xpn.xwiki.XWikiContext)</old>
                 </item>
+                <item>
+                  <criticality>allowed</criticality>
```

Comment:
That's not good, you put your breaking justification under my own justification. You need to add a `revapi.differences` block here and provide your own explanation.